src/CocktailDetails/CocktailDetails.tsx

```diff
@@ -1,66 +1,125 @@
+  };
+
+  useEffect(() => {
+    getCocktail();
+  }, []);
 	//modify lint file?
 	
-	return (
-		<section className='cocktail-details-wrapper'>
-			<section className='cocktail-details-card'>
-				<h3>{cocktailInfo.strDrink}</h3>
-				<img className='COTD-img' src={`${cocktailInfo.strDrinkThumb}`} alt={`${cocktailInfo.strDrink}`} /> 
-				<section className='info-wrapper'>
-				<p className='detail-paragraph'><div className='sub-heading'>Type:  </div> {cocktailInfo.strCategory}, {cocktailInfo.strAlcoholic}</p>
-				<p className='detail-paragraph'><div className='sub-heading'>Glass:  </div>{cocktailInfo.strGlass}</p>
-				<h4>Instructions:</h4>
-					<p>{cocktailInfo.strInstructions}</p>
-				<div className='sub-heading'>Ingredients:</div>
-				<section className='ingredient-container'>
-					<ul>
-						{displayIngredients(13, 'strIngredient').map((item, i) => <li key={i}>{item}</li>)}
-					</ul>
-					<ul className='ingr-measure-list'>
-						{displayIngredients(10, 'strMeasure').map((item, i) => <li key={i}>{item}</li>)}
-					</ul>
-				</section>
-				</section>
-			</section>
-		</section>	
-	)
-}
+	const clickHandler = () => {
+		setIsFavorite(!isFavorite)
+		toggleFavorites(id)
+	}
+
+  return (
+    <section className="cocktail-details-wrapper">
+      <section className="cocktail-details-card">
+        <h3>{cocktailInfo.strDrink}</h3>
+        <img
+          className="COTD-img"
+          src={`${cocktailInfo.strDrinkThumb}`}
+          alt={`${cocktailInfo.strDrink}`}
+        />
+        <section className="info-wrapper">
+          <p className="detail-paragraph">
+            <div className="sub-heading">Type: </div> {cocktailInfo.strCategory}
+            , {cocktailInfo.strAlcoholic}
+          </p>
+          <p className="detail-paragraph">
+            <div className="sub-heading">Glass: </div>
+            {cocktailInfo.strGlass}
+          </p>
+          <h4>Instructions:</h4>
+          <p>{cocktailInfo.strInstructions}</p>
+          <div className="sub-heading">Ingredients:</div>
+          <section className="ingredient-container">
+            <ul>
+              {displayIngredients(13, "strIngredient").map((item, i) => (
+                <li key={i}>{item}</li>
+              ))}
+            </ul>
+            <ul className="ingr-measure-list">
+              {displayIngredients(10, "strMeasure").map((item, i) => (
+                <li key={i}>{item}</li>
+              ))}
+            </ul>
+          </section>
+          {!isFavorite && (
+            <button
+              type="button"
+              aria-label="add-to-favorites"
+              onClick={() => clickHandler()}
+            >
+              Add to Favorites
+            </button>
+          )}
+          {isFavorite && (
+            <button
+              type="button"
+              aria-label="add-to-favorites"
+              onClick={() => clickHandler()}
+            >
+              Remove from Favorites
+            </button>
```

Comment:
You could dry up this section by just conditionally rendering different text, rather than 2 different buttons.